test/phonon.jl

```diff
@@ -3,10 +3,11 @@ using Test
 using DFTK
 using DFTK: energy_forces_pairwise
 using LinearAlgebra
-using ForwardDiff
-using StaticArrays
 using Random
 
```

Comment:
I would propose to have this in a file `phonon/main.jl` instead.

Reply:
Or instead have a `phonon/runtests.jl` which has a similar stucture than the main runtests, e.g. everything is encapsulated by `@testset`s or so ... this also makes sure that things like `unpack` don't bleed out and potentially pollute other tests.